src/modules/commands/fight.js

```diff
@@ -0,0 +1,22 @@
+const isEmpty = require('lodash/isEmpty');
+const random = require('lodash/sample');
+const command = require('../command.filter');
+
+const fights = async function (request, context) {
+    const { i18n, send } = context;
+    const { args: opponent } = request;
```

Comment:
учим javascript! ты переименовала переменную... 
```suggestion
    const { args: { opponent } } = request;
```
разницу понимаешь?